### ISSUE TITLE
Instructor Interactive Rainbowgrades 

### ISSUE THREAD

**Issue by ziesski** [Submitty/Submitty#10273]
<img width="1133" alt="스크린샷 2024-03-22 오후 5 04 18" src="https://github.com/Submitty/Submitty/assets/123261952/06df9793-b53e-4f3a-8f6b-5aa04776af42">

Instructor has big rainbowgrades sheet, however, when scrolling both vertically and horizontally, the title row and column does not follow the scroll. If the sheet is big enough, instructor may get lost which student they were looking, or which gradeable it was. In order to fix it I think we should make the first row and column follow the scroll. 

<img width="750" alt="스크린샷 2024-03-22 오후 5 04 41" src="https://github.com/Submitty/Submitty/assets/123261952/69885777-5650-4c1b-914b-ee19ea39c618">

**Comment by sadam-04** [comment]
Wouldn't this be a change to the rainbowgrades repo?

**Comment by ziesski** [comment]
> Wouldn't this be a change to the rainbowgrades repo?

Yes, but all Submitty related issues are collected in Submitty/Submitty, and we use tag to categorize them.

**Comment by JWei17** [comment]
How can I look at the rainbow grade sheet as a instructor?

**Comment by SRoar** [comment]
Hi, I would like to work on this issue. I plan to start with the sticky header/column fix for the Rainbow Grades table view, then add the make section sort option to the configuration GUI. Please let me know if there are any implementation guidelines I should follow.
